source/_posts/2021-02-03-release-20212.markdown

```diff
@@ -13,29 +13,38 @@ og_image: /images/blog/2021-02/social.png
 
 <img src='/images/blog/2021-02/social.png' style='border: 0;box-shadow: none;'>
 
-These are the beta release notes for Home Assistant Core 2021.2 (and is thus a
-work in progress).
+Dear homie,
```

Comment:
It's a bit tight.

<img width="158" alt="image" src="https://user-images.githubusercontent.com/1444314/106768579-554b4800-663c-11eb-8b0c-5d2771257575.png">

Reply:
Done